src/views/search/utils.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { TFunction } from 'i18next';
+import { v4 as uuid } from 'uuid';
+
+import { ContactInputItem, KeywordState, Query, SearchQueryItem } from '../../types';
+
+type EmptyListMessages = { title: string; description: string }[];
+
+export const EmptyListMessages = (t: TFunction): EmptyListMessages => [
+	{
+		title: t('displayer.search_title1', 'Start another search'),
+		description: t(
+			'displayer.search_description1',
+			'Or select “Advanced Filters” to refine your search.'
+		)
+	},
+	{
+		title: t('displayer.search_title2', 'We’re sorry but there are no results for your search'),
+		description: t('displayer.search_description2', 'Try to start another search.')
+	},
+	{
+		title: t('displayer.search_title3', 'There are no results for your search.'),
+		description: t(
+			'displayer.search_description3',
+			'Check the spelling and the filters options or try with another keyword.'
+		)
+	}
+];
+
+export const EmptyFieldMessages = (t: TFunction): EmptyListMessages => [
+	{
+		title: t(
+			'displayer.search_title4',
+			'Select one or more results to perform actions or display details.'
+		),
+		description: ''
+	}
+];
+
+function getRegex(prefix?: string): RegExp {
+	return new RegExp(`^${prefix}:.*`, 'i');
+}
+
+export function getChipString(item: SearchQueryItem | ContactInputItem, prefix: string): string {
+	const regex = getRegex(prefix);
+	let resultString = '';
+	if ((item as SearchQueryItem).label) {
+		resultString = (item as SearchQueryItem).label;
+	}
+	if ((item as ContactInputItem).fullName) {
+		resultString = (item.label || (item as ContactInputItem).fullName) ?? '';
```

Comment:
The fullName is never used if the label is defined: is it the expected behaviour?

Reply:
yes, that is correct